### PR TITLE
Remember drop down value (EDS-548) and send to query

### PR DESCRIPTION
**Pull request by goulter**
- Remember drop down value, closes EDS-548
- Discovered and fixed bug where query drop down value was not being updated/sent when user does not change the selected drop down value.

Including @jdiverp for visibility

### CODE REVIEW

**Comment by goulter** [comment]
I admit `dynamic_search_options.html` and `search_options.html` is a little tedious but, this is done to stay with Flask as much as possible.

File: husky_directory/static/js/setSearchNameAttribute.js [hunk 1]
```diff
@@ -8,9 +8,16 @@
...
+
+function updateSearchSelect(){
+    var value = document.getElementById("method").value;
     searchFormField.setAttribute("name", value);
 }
+
+//function searchNoChange(){
+//    var value = document.getElementById("method").value;
+//    searchFormField.setAttribute("name", value);
+//}
```
Comment (tomthorogood) [inline husky_directory/static/js/setSearchNameAttribute.js:23]: Can you remove this?

**Review by tomthorogood — APPROVED** [review]
Just one request to remove some cruft. We shouldn't introduce commented code.

File: husky_directory/static/js/setSearchNameAttribute.js (hunk 1 above)
Comment (goulter) [reply husky_directory/static/js/setSearchNameAttribute.js:23]: woops!

**Comment by tomthorogood** [comment]
btw, this looks great.

**Comment by tomthorogood** [comment]
> btw, this looks great.

code-wise i mean 👍

**Review by tomthorogood — COMMENTED** [review]
oops, forgot to approve this before merging :)

**Pull request merged**